pylint/checkers/exceptions.py

```diff
@@ -337,6 +369,38 @@ def _check_bad_exception_context(self, node):
         ):
             self.add_message("bad-exception-context", node=node)
 
+    def _check_raise_missing_from(self, node: astroid.Raise) -> None:
+        assert node.cause is None
```

Comment:
Please remove this, we already asserted that `node.cause` is not empty in the call site of this function.

Reply:
Fixed.